inc/saiswitch.h

```diff
@@ -739,111 +739,119 @@ typedef enum _sai_switch_attr_t
      * @type sai_int32_t
      * @flags READ_ONLY
      */
-    SAI_SWITCH_ATTR_AVERAGE_TEMP,
+    SAI_SWITCH_ATTR_AVERAGE_TEMP = SAI_SWITCH_ATTR_START + 0x16,
 
     /**
      * @brief Minimum priority for ACL table
      *
      * @type sai_uint32_t
      * @flags READ_ONLY
      */
-    SAI_SWITCH_ATTR_ACL_TABLE_MINIMUM_PRIORITY,
+    SAI_SWITCH_ATTR_ACL_TABLE_MINIMUM_PRIORITY = SAI_SWITCH_ATTR_START + 0x17,
 
     /**
      * @brief Maximum priority for ACL table
      *
      * @type sai_uint32_t
      * @flags READ_ONLY
      */
-    SAI_SWITCH_ATTR_ACL_TABLE_MAXIMUM_PRIORITY,
+    SAI_SWITCH_ATTR_ACL_TABLE_MAXIMUM_PRIORITY = SAI_SWITCH_ATTR_START + 0x18,
 
     /**
      * @brief Minimum priority for ACL entry
      *
      * @type sai_uint32_t
      * @flags READ_ONLY
      */
-    SAI_SWITCH_ATTR_ACL_ENTRY_MINIMUM_PRIORITY,
+    SAI_SWITCH_ATTR_ACL_ENTRY_MINIMUM_PRIORITY = SAI_SWITCH_ATTR_START + 0x19,
 
     /**
      * @brief Maximum priority for ACL entry
      *
      * @type sai_uint32_t
      * @flags READ_ONLY
      */
-    SAI_SWITCH_ATTR_ACL_ENTRY_MAXIMUM_PRIORITY,
+    SAI_SWITCH_ATTR_ACL_ENTRY_MAXIMUM_PRIORITY = SAI_SWITCH_ATTR_START + 0x1a,
```

Comment:
you dont need to use SAI_SWITCH_ATTR_START , you can just use explicit value, ps why you need to reorder everything ?